raiden-ts/src/utils/rx.ts

```diff
@@ -66,23 +68,32 @@ export function distinctRecordValues<R>(
 /**
  * Operator to repeat-subscribe an input observable until a notifier emits
  *
- * @param notifier - Notifier observable
- * @param delayMs - Delay between retries
- * @returns Monotype operator
+ * @param notifier - Notifier observable to stop repeating
+ * @param delayMs - Delay between retries; in milliseconds, or an Iterator of delays
```

Comment:
Both are in ms right?
```suggestion
 * @param delayMs - Delay between retries or an Iterator of delays; in milliseconds
```